certification/pyxis/pyxis.go

```diff
@@ -76,17 +76,13 @@ func (p *pyxisEngine) createImage(ctx context.Context, certImage *CertImage) (*C
 	return &newCertImage, nil
 }
 
-func (p *pyxisEngine) createRPMManifest(ctx context.Context, imageId string, rpms []RPM) error {
-	rpmManifest := RPMManifest{
-		ImageID: imageId,
-		RPMS:    rpms,
-	}
+func (p *pyxisEngine) createRPMManifest(ctx context.Context, rpmManifest *RPMManifest) error {
 	b, err := json.Marshal(rpmManifest)
 	if err != nil {
 		log.Error(err)
 		return err
 	}
-	req, err := newRequestWithApiToken(ctx, http.MethodPost, getPyxisUrl(fmt.Sprintf("images/id/%s/rpm-manifest", imageId)), p.ApiToken)
+	req, err := newRequestWithApiToken(ctx, http.MethodPost, getPyxisUrl(fmt.Sprintf("images/id/%s/rpm-manifest", rpmManifest.ImageID)), p.ApiToken)
```

Comment:
When we write rpm manifest to disk we do not have the image id, so we pass it somewhere.